### PR TITLE
Fix: cv_validate_v3 broken with cvprac 1.4.0 due to argument keyword change

### PR DESCRIPTION
**Pull request by noredistribution**
## Change Summary

cvprac updated the keyword arguments for `validate_config_for_device()` from `device_mac` to `dev_mac` in https://github.com/aristanetworks/cvprac/commit/69fd313b18fdd386d2b3e8cf087d5b27d78cb923#diff-ff77c7a073e87f20dd555db78c0b2ee12ed8924d0cc12645fb2e33034ec523aaL1359 and playbooks are failing with:

```
    Traceback (most recent call last):
      File "/home/pprad133/.ansible/tmp/ansible-local-175841olddsl1/ansible-tmp-1716569183.2255661-17610-80870313378101/AnsiballZ_cv_validate_v3.py", line 107, in <module>
        _ansiballz_main()
      File "/home/pprad133/.ansible/tmp/ansible-local-175841olddsl1/ansible-tmp-1716569183.2255661-17610-80870313378101/AnsiballZ_cv_validate_v3.py", line 99, in _ansiballz_main
        invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
      File "/home/pprad133/.ansible/tmp/ansible-local-175841olddsl1/ansible-tmp-1716569183.2255661-17610-80870313378101/AnsiballZ_cv_validate_v3.py", line 47, in invoke_module
        runpy.run_module(mod_name='ansible_collections.arista.cvp.plugins.modules.cv_validate_v3', init_globals=dict(_module_fqn='ansible_collections.arista.cvp.plugins.modules.cv_validate_v3', _modlib_path=modlib_path),
      File "/usr/lib/python3.10/runpy.py", line 224, in run_module
        return _run_module_code(code, init_globals, run_name, mod_spec)
      File "/usr/lib/python3.10/runpy.py", line 96, in _run_module_code
        _run_code(code, mod_globals, init_globals,
      File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
        exec(code, run_globals)
      File "/tmp/ansible_arista.cvp.cv_validate_v3_payload_igauvor3/ansible_arista.cvp.cv_validate_v3_payload.zip/ansible_collections/arista/cvp/plugins/modules/cv_validate_v3.py", line 163, in <module>
      File "/tmp/ansible_arista.cvp.cv_validate_v3_payload_igauvor3/ansible_arista.cvp.cv_validate_v3_payload.zip/ansible_collections/arista/cvp/plugins/modules/cv_validate_v3.py", line 153, in main
      File "/tmp/ansible_arista.cvp.cv_validate_v3_payload_igauvor3/ansible_arista.cvp.cv_validate_v3_payload.zip/ansible_collections/arista/cvp/plugins/module_utils/validate_tools.py", line 194, in manager
    TypeError: CvpApi.validate_config_for_device() got an unexpected keyword argument 'device_mac'
  module_stdout: ''
  msg: |-
    MODULE FAILURE
    See stdout/stderr for the exact error
  rc: 1
```

Thanks for @pmprado for reporting

## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.cvp.cv_validate_v3`

## Proposed changes

removed the keyword argument name and just passing the vars

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x ] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly. (check the box if not applicable)
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by pmprado — APPROVED** [review]
Looks great and worked fine testing in a production environment.

**Review by mharista — APPROVED** [review]

**Review by sugetha24 — APPROVED** [review]
LGTM!

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -62,7 +62,7 @@ repos:
           - '<!--|  ~|  -->'
 
   - repo: https://github.com/pycqa/pylint
-    rev: "v2.16.1"
+    rev: "v3.1.1"
```
Comment (carlbuchmann) [inline .pre-commit-config.yaml]: revert this change
```suggestion
    rev: "v2.16.1"
```

**Review by carlbuchmann — APPROVED** [review]
LGTM
